lib/application.rb

```diff
@@ -1,12 +1,220 @@
 class LibraryApp
```

Comment:
Try to use same name for file and class. 
Also, please split this file. It's too long (should be <100 mostly)